assignment4/sass/pages/_details.scss

```diff
@@ -0,0 +1,348 @@
+$report-bg-color : #ffffff;
+
+.middle-container {
+    width: 82%;
+    float: right;
+	.page-status {
+    	background-color: $report-bg-color;
+    	margin: 5px;
+    	padding-bottom:15px;
+    	padding-top:10px;
+		span {
+			margin-left:20px;
+		}
+	}
+	.report {
+		background-color: $report-bg-color;
+	    margin: 5px;
+	}
+	.test-report {
+		.header {
+    		padding-bottom:40px;
+			.header-text {
+				float: left;
+	    		width: 35%;
+	    		margin-left:20px;
+			}
+
+			h5 {
+				color:blue;
+			}
+
+			button {
+				float:left;
+	    		margin: 10px 20px 5px 20px;
+	    		background-color:#624b3d;
+	    		color:white;
+	    		padding:5px;
+	    		border-color:#624b3d;
+			}
+
+			.subscription {
+				float: right;
+			    margin-right: 30px;
+			    color: red;
+			    margin-top: 8px
+			}
+		}
+		.detail {
+		    width: 100%;
+    		.report-img {
+			    width: 45%;
+			    text-align:center;
+				img {
+			        width: 20%;
+				}
+    		}
+    		
+    	
+		}
+
+		.user-detail {
+			width: 50%;
+			float:right;
+			margin-top:-130px;
+			label {
+				color:blue;
+			}
+			input[type="text"] {
+			    border-left: none;
+			    border-right: none;
+			    border-top: none;
+			    display: block;
+			    margin-bottom: 20px;
+			    width: 91%;
+			    border-bottom: 2px solid;
+			    border-color: blue;
+    		}
+
+			button {
+				float: right;
+			    margin-right: 44px;
+			    background-color: #7E90e0;
+			    padding: 8px;
+			    border-color: #7e90e0;
+			    margin-top: -10px;
+			}
+			a {
+				text-decoration: none;
+			}
+		}
+		.user-name {
+		    margin-top: 15px;
+			width: 100%;
+			font-size: 14px;
+			span {
+				padding: 4px;
+			    background-color: #868789;
+			    border-radius: 23px;
+			    margin-right: 15px;
+			    margin-bottom: 10px;
+			    padding-right:0px;
+			    span {
+		    		background-color: red;
+				    padding: 1px;
+				    padding-left: 6px;
+				    margin-left: 5px;
+			    }
+
+			}
+			a {
+				text-decoration: none;
+			    color: white;
+			    font-size:10px;
+
+			}
+		}
+		
+
+	}
+
+	.test-report-2 {
+		@extend .test-report;
+		.user-detail {
+			label {
+				color:grey;
+			}
+			input[type="text"] {
+				border-color:grey;
+			}
+		}
+	}
+
+	.tag-summary {
+		@extend .test-report-2;
+	}
+
+}
+
+@media only screen and (max-width:1294px) and (min-width:1234px) {
+	.middle-container {
+		width:81.1%;
+	}
+	.detail {
+		padding-top:12px;
+	}
+	.user-detail {
+		margin-top:-123px;
+	}
+
+}
+
+@media only screen and (max-width:1233px) and (min-width:1167px) {
```

Comment:
@priyanka1110  Actually sidebar is fixed. So when I am decreasing the size of screen. middle container is going down.  That is what i need to add lot of media query.